asciidoc/modules/con_mg_changes-in-vertx-jdbc-client.adoc

```diff
@@ -0,0 +1,8 @@
+[id="changes-in-vertx-jdbc-client_{context}"]
+= Changes in {VertX} JDBC client
+
+The following section describes the changes in {VertX} JDBC client.
+
+== Merged Vert.x SQL common module in JDBC client module
+
+The `io.vertx:vertx-sql-common` module has been merged in `io.vertx:vertx-jdbc-client` module. You will have to remove the `io.vertx:vertx-sql-common` dependency file because `io.vertx:vertx-jdbc-client` will include it.
```

Comment:
@vietj @jponge @pmlopes Do we need to anything from the following JDBC-client adoc?

https://github.com/vert-x3/vertx-4-migration-guide/blob/master/asciidoc/jdbc_client.adoc#jdbc-client

Reply:
I this document bullet points represent a good summary of changes so you can probably take content from here.

Reply:
there is also an open issue https://github.com/vert-x3/vertx-4-migration-guide/issues/1 because we need a real section